### ISSUE TITLE
Windows desktop app opens Okta screen, but never redirects to login

### ISSUE THREAD

**Issue by miro444** [mattermost/desktop#2405]
#### Summary
Unable to login to MM windows desktop app via Okta

#### Steps to reproduce
Get Okta integrated with the onprem MM. VErsion 5.2.0 and 5.2.1 

#### Expected behavior
MM app is asking you to login, so i put in https for oour onprem server and click to continue. AFter that, I get normal "Login to Okta" screen. When clicking the button, nothing happens. 

#### Observed behavior (that appears unintentional) 
As described above. 

#### Possible fixes
Tried all of the suggestions here: https://docs.mattermost.com/install/desktop-app-install.html 
None helped. 

![image](https://user-images.githubusercontent.com/105832553/202927854-bd9cbbef-e31a-43dc-b707-443fa2edc07d.png)

**Comment by miro444** [comment]
WHat kind of log are you looking for? There is like 10 tabs

**Comment by miro444** [comment]
NEtwork does not log anything btw. THe button "login to okta" simply does not do anything.

**Comment by miro444** [comment]
There was nothing on Console tab either. I will see if we can do server restart at some point and will update the desktop app again to the latest.

**Comment by devinbinnie** [comment]
@miro444 Any luck with server restart or app update?

**Comment by miro444** [comment]
Hey guys, 
Restart did not fix anything. And I tried updating, it went to 5.2.x, but after shutting it down and trying to restart, it threw the JAva error below and failed to load. 

![image](https://user-images.githubusercontent.com/105832553/204911931-45244285-d6d4-493c-9048-5d538f4fb49c.png)

SO i had to uninstall it, then reinstalled with mattermost-desktop-5.1.1-x64.msi. Not working fine again. So downloaded manually mattermost-desktop-5.2.2-rc.2-x64.msi and problem with Okta button is back. 

THe butonn has this link: https://mattermost.xxxxxxxxxxxx/login/sso/saml, but does again nothing. 

Console shows err, but it was there after I opened it and nothing is generated when the button is clicked as it does not click. 

![image](https://user-images.githubusercontent.com/105832553/204913235-8d628ac9-a2c2-4c2a-a4b5-0dec6f4e62b0.png)
